doc/man1/openssl-verification-options.pod

```diff
@@ -548,10 +569,14 @@ This option has no effect and is retained for backward compatibility only.
 
 =head2 Certificate Extensions
 
-Options like B<-purpose> lead to checking the certificate extensions,
+Options like B<-purpose> lead to checking specific certificate extensions,
 which determine what the target certificate and intermediate CA certificates
```

Comment:
```suggestion
which determine what the end-entity certificate and intermediate CA certificates
```

Reply:
Since  this holds even for TA certs, changed to simply: 
`which determine what certificates can be used for.`